modules/misunderstood/src/views/full/index.tsx

```diff
@@ -307,6 +309,53 @@ export default class MisunderstoodMainView extends React.Component<Props, State>
     })
   }
 
+  clickImportData = () => {
+    this.importInput.current.click()
+  }
+
+  importData = evt => {
+    try {
+      let files = evt.files
+      if (!files.length) {
+        return
+      }
+      let file = files[0]
+      // TODO: make this a generic filereader, we probably use this multiple times
+      let reader = new FileReader()
```

Comment:
There are 3 'let' which are not reassigned, please use const instead. Optional chaining can be used when accessing index, ex:
```js
const file = evt.files?.[0]
if(!file){
  return 
}
``